js/samples/04.ai.a.teamsChefBot/teamsapp.local.yml

```diff
@@ -41,10 +41,19 @@ provision:
       appPackagePath: ./build/appPackage/appPackage.${{TEAMSFX_ENV}}.zip # Relative path to this file. This is the path for built zip file.
 
 deploy:
+ # Install any dependencies and build the web app using NPM
+  - uses: cli/runNpmCommand
+    name: install dependencies
+    with:
+      args: install --no-audit
+
```

Comment:
Do we also need to have `npm run build`?

Reply:
Not for a local run